File: Serie/src/main/java/com/ferrazeduardo/escola/Serie/domain/ValorSerie.java
package com.ferrazeduardo.escola.Serie.domain;

import jakarta.persistence.*;

import java.math.BigDecimal;
import java.util.UUID;

public class ValorSerie {

    public ValorSerie(BigDecimal VL_SERIE, String DS_TURNO) {
        this.VL_SERIE = VL_SERIE;
        this.DS_TURNO = DS_TURNO;
    }

    private BigDecimal VL_SERIE;
    private String DS_TURNO;


}

File: Serie/Serie.Domain/Entity/ValorSerie.cs
using System;

namespace Serie.Domain.Entity;

public class ValorSerie : SeedWorks.Entity
{

    public ValorSerie()
    {
        
    }

    public ValorSerie(Decimal valor,String turno,Serie serie)
    {
        Serie = serie;
        VL_Serie = valor;
        DS_TURNO = turno;
    }

    public Serie Serie { get; private set; }
    public Decimal VL_Serie { get; private set; }
    public String DS_TURNO { get; private set; }
}
